File: trackpoint-frontend/src/type/data/stat/project.ts

/**
 * 项目统计中的单个项目
 */
export interface IProjectStat {
    id: string
    name: string
    /**
     * 默认事件数量
     */
    default_count: number
    /**
     * 自定义事件数量
     */
    custom_count: number
    /**
     * 记录数量
     */
    record_count: number
    /**
     * 客户端数量
     */
    client_count: number
}

File: trackpoint-backend/domain/vo/data/stat/project.py




from pydantic import BaseModel


class ProjectStatVO(BaseModel):
    """项目统计中响应体中的单个项目"""
    id: str
    name: str
    default_count: int
    """默认事件数量"""
    custom_count: int
    """自定义事件数量"""
    record_count: int
    """记录数量"""
    client_count: int
    """客户端数量"""
